src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -482,9 +482,12 @@ public enum OptimizationAlgorithmType { GradientDescent, AcceleratedGradientDesc
         /// <summary>
         /// Early stopping rule. (Validation set (/valid) is required).
         /// </summary>
+        [BestFriend]
         [Argument(ArgumentType.Multiple, HelpText = "Early stopping rule. (Validation set (/valid) is required.)", ShortName = "esr", NullName = "<Disable>")]
```

Comment:
Add attribute Name = "EarlyStoppingRule"  this would keep core_manifest.json same and will avoid big changes in NimbusML

Reply:
Fixed but I'd expect tons of breaking changes to entry-point had happened somewhere.

---
In reply to: [262650819](https://github.com/dotnet/machinelearning/pull/2851#discussion_r262650819) [](ancestors = 262650819)